src/Segmentation/segmented-image-utilities.jl

```diff
@@ -69,3 +71,75 @@ Find pixels in a segmented image with non-zero labels
 function binarize_segments(segments::SegmentedImage)::AbstractArray{Gray}
     return Gray.(labels_map(segments) .> 0)
 end
+
+
+"""stitch_clusters(tiles, segmented_image, minimum_overlap, grayscale_threshold)
+
+Stitches clusters across tile boundaries based on neighbor with largest shared boundary.
+The algorithm finds all pairs of segment labels at the tile edges. Then, we count the number of 
+times each right-hand label is paired to a left-hand label, and for pairs with at least `minimum_overlap` pixel overlap,
+the right-hand label is assigned as a candidate pair to the left-hand label. If the difference in grayscale
+intensity is less than `grayscale_threshold`, the objects are merged. The function returns an image index map.
+
+"""
+function stitch_clusters(segmented_image, tiles, minimum_overlap=4, grayscale_threshold=0.1) 
+    grayscale_magnitude(c) = Float64(Gray(c))
+    
+    idxmap = deepcopy(segmented_image.image_indexmap)
```

Comment:
Is this the same as 
```suggestion
    idxmap = deepcopy(labels
_map(segmented_image))
```
?